core/node/eth_sender/src/eth_fees_oracle.rs

```diff
@@ -8,7 +8,9 @@ use zksync_eth_client::{ClientError, EnrichedClientError};
 use zksync_node_fee_model::l1_gas_price::TxParamsProvider;
 use zksync_types::eth_sender::TxHistory;
 
-use crate::{abstract_l1_interface::OperatorType, EthSenderError};
+use crate::{
+    abstract_l1_interface::OperatorType, EthSenderError, EthSenderError::ExceedMaxBaseFee,
```

Comment:
Bikeshedding: Please don't import enum variants into the scope; it makes the code more difficult to read.

Reply:
Can you clarify the alternative?

Reply:
Use the qualified variant name (`EthSenderError::ExceedMaxBaseFee` in this case):

```rust
return Err(EthSenderError::ExceedMaxBaseFee);
```

Reply:
thanks 👍